stingray/modeling/posterior.py

```diff
@@ -139,8 +140,8 @@ def logprior(t0, neg=False):
     return logprior
 
 
+@six.add_metaclass(abc.ABCMeta)
```

Comment:
No troubles, but the syntax has changed between Py2 and Py3, so `six` takes care of it. https://pythonhosted.org/six/#six.add_metaclass